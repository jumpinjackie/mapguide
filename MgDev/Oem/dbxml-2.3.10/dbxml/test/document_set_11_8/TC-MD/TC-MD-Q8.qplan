<XQuery>
  <DbXmlNav>
    <QueryPlanFunction result="collection" container="11.8.3.dbxml">
      <OQPlan>n(P(node-element-presence-none,=,author),P(node-element-equality-string,prefix,name),V(node-attribute-equality-string,@id,=,'2'))</OQPlan>
    </QueryPlanFunction>
    <DbXmlStep axis="child" name="article" nodeType="element"/>
    <DbXmlFilter>
      <DbXmlCompare name="equal" join="attribute" name="id" nodeType="attribute">
        <Sequence>
          <AnyAtomicTypeConstructor value="2" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
        </Sequence>
      </DbXmlCompare>
    </DbXmlFilter>
    <DbXmlStep axis="child" name="prolog" nodeType="element"/>
    <DbXmlStep axis="child" uri="*" name="*" nodeType="element"/>
    <DbXmlStep axis="child" name="author" nodeType="element"/>
    <DbXmlStep axis="child" name="name" nodeType="element"/>
  </DbXmlNav>
</XQuery>
