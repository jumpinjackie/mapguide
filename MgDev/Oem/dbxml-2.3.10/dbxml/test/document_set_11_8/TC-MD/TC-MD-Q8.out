<name>Poorani Yuguchi</name><name>Hilding Krackhardt</name><name>Sumanta Svjatnyi</name><name>Kwok Encarnacion</name><name>Fanya Korncoff</name>